diff --git a/CMakeLists.txt b/CMakeLists.txt
index 17484c1..8f64de7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -422,14 +422,12 @@ endif()
 #--------------------------------------------------------------------
 include (ExternalProject)
 
+find_package(cereal REQUIRED)
+
 # third party directories
 set( THIRDPARTYDIR ${CMAKE_CURRENT_SOURCE_DIR}/third-party )
 include_directories( ${THIRDPARTYDIR}/include )
 
-### Handle third-party CEREAL
-include_directories( ${THIRDPARTYDIR}/cereal/include )
-install(DIRECTORY ${THIRDPARTYDIR}/cereal/include/ DESTINATION include/palisade)
-
 include_directories( third-party/google-test/googletest third-party/google-test/googletest/include )
 include_directories( ${CMAKE_CURRENT_BINARY_DIR}/third-party/include )
 include_directories( ${CMAKE_CURRENT_BINARY_DIR}/src/core )
