diff --git a/CMakeLists.txt b/CMakeLists.txt
index 32f9116..bca05dc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.9)
 project(tfhe++ CXX)
 set(CMAKE_CXX_COMPILER g++)
-set(CMAKE_CXX_FLAGS "-march=native -O3 -std=c++17 -g")
+set(CMAKE_CXX_FLAGS "-std=c++17")
 
 OPTION(USE_80BIT_SECURITY "Use 80bit security parameter(faster)" OFF)
 OPTION(ENABLE_TEST "Build tests" OFF)
